astro/src/components/Card.astro

```diff
@@ -0,0 +1,78 @@
+---
+import Carousal from './Carousel.astro';
+// export interface Props {
+// 	title: string;
+// 	body: string;
+// 	href: string;
+// }
+
+// const { href, title, body } = Astro.props;
+---
+
+<li class="link-card" style="align-items: center; display: flex; flex-direction: column; ; gap: 1rem;">
+	
+  <Carousal/>
+  <div>
+	<span style="padding: 0.8rem 1rem; font-size: 1rem;">
+		Theft Date
```

Comment:
here it is possible to use the `t()` function in order to achieve dynamic translations:
`{ t('components.card.theft_date') }`